### PR TITLE
LIME-159: Created frontend skeleton and added common express lib

### PR DESCRIPTION
**Pull request by gds-jbentham**
### What changed

Added the common express lib and aligned more closely with the common cri pattern

### Why did it change

To allow communication with the new Driving Licence CRI backend

### Issue tracking

- [LIME-159](https://govukverify.atlassian.net/browse/LIME-159)

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -1,70 +1,130 @@
...
-    client: dynamodb,
-    table: SESSION_TABLE_NAME,
-  });
-}
+const { setup } = require("hmpo-app");
 
 const loggerConfig = {
   console: true,
-  consoleJSON: true,
+    consoleLevel: 'debug',
```
Comment (ElliotMurphyGDS) [inline src/app.js:34]: I'm sure you're aware but will probably need to be put back to info

File: src/app/drivingLicence/controllers/root.js [hunk 2]
```diff
@@ -6,7 +6,7 @@ var logger = require('hmpo-logger').get();
 class RootController extends BaseController {
   async saveValues(req, res, next) {
     const sharedClaims = req.session?.shared_claims;
-    console.log('JB!@£$%^&*SharedClaims' + sharedClaims)
+    logger.info('JB!@£$%^&*SharedClaims' + sharedClaims)
```
Comment (ElliotMurphyGDS) [inline src/app/drivingLicence/controllers/root.js:9]: Again probably pointing out the obvious but feel free to get rid of my weird logging

File: src/app/drivingLicence/controllers/validate.js [hunk 3]
```diff
@@ -1,23 +1,24 @@
...
+
     const firstName = req.sessionModel.get("firstName");
     const middleNames = req.sessionModel.get("middleNames");
     const forenames =
       middleNames === ""
         ? firstName.split(" ")
         : firstName.split(" ").concat(middleNames.split(" "));
     const attributes = {
-      documentNumber: req.sessionModel.get("documentNumber"),
+      drivingLicenceNumber: req.sessionModel.get("drivingLicenceNumber"),
```
Comment (ElliotMurphyGDS) [inline src/app/drivingLicence/controllers/validate.js:20]: Feel free to ignore my comment on the backend PR looks like we're now using drivingLicenceNumber

Comment (gds-jbentham) [reply src/app/drivingLicence/controllers/validate.js:20]: Needed to to make it play nice with the frontend

**Review by ElliotMurphyGDS — DISMISSED** [review]

**Review by ElliotMurphyGDS — APPROVED** [review]

**Review by smsgds — APPROVED** [review]

**Pull request merged**